pkg/resources/containers.go

```diff
@@ -40,6 +40,12 @@ const qRadarConfigKey = "remoteSyslog.conf"
 const AuditLoggingCertName = "fluentd"
 const AuditPolicyControllerDeploy = "audit-policy-controller"
 
+const defaultImageRegistry = "hyc-cloud-private-edge-docker-local.artifactory.swg-devops.com/ibmcom-amd64"
+const defaultFluentdImageName = "fluentd"
+const defaultFluentdImageTag = "1.6.2-ubi7"
```

Comment:
missing the `v`

Reply:
Also, ubi7 hasn't been merged to master so this would cause an error pulling the image